sdk/browser/test/integration/otp/index.test.ts

```diff
@@ -39,174 +30,154 @@ const credentialShareRequestToken =
   '50b739ac0e5eb4add1961c88d9f0486b37be928bccf2b19fb5a1d2b7c9bbe'
 
 const options: __dangerous.SdkOptions = getOptionsForEnvironment()
+const { env } = getOptionsForEnvironment(true)
 
-describe('AffinityWallet (flows that require OTP)', () => {
-  it('Save Open Attestation credential and #deleteCredential scenario', async () => {
-    const cognitoUsername = generateEmail()
+const messageParameters: MessageParameters = {
+  message: `Your verification code is: {{CODE}}`,
+  subject: `Verification code`,
+}
 
-    const token = await AffinityWallet.signUp(cognitoUsername, cognitoPassword, options)
+const waitForOtpCode = async (inbox: TestmailInbox): Promise<string> => {
+  const { body } = await inbox.waitForNewEmail()
+  return body.replace('Your verification code is: ', '')
+}
 
-    await wait(DELAY)
-    const signUpOtp = await getOtp()
+const createInbox = () => new TestmailInbox({ prefix: env, suffix: 'otp.browser' })
 
-    const networkMember = await AffinityWallet.confirmSignUp(token, signUpOtp, options)
+describe('AffinityWallet [OTP]', () => {
+  it('Save Open Attestation credential and #deleteCredential scenario', async () => {
+    const inbox = createInbox()
+    const password = COGNITO_PASSWORD
 
-    let credentials
+    const signUpToken = await AffinityWallet.signUp(inbox.email, password, options, messageParameters)
```

Comment:
By the way, since we're here: @DenisPopov15 , do we really want to use `TEST_SECRETS.COGNITO_PASSWORD` here instead of generating a new password every time? 🤔
(This code was there for a long time, so the comment is not related to this PR)

Reply:
Created a ticket: https://replika.atlassian.net/browse/FTL-1563